pkgs/development/python-modules/napari-console/default.nix

```diff
@@ -1,29 +1,50 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, setuptools-scm
-, pytestCheckHook
-, pytest
-, ipython
+, imageio
 , ipykernel
-, qtconsole
+, ipython
 , napari-plugin-engine
-, imageio
-}: buildPythonPackage rec {
+, pythonOlder
+, qtconsole
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
   pname = "napari-console";
-  version = "0.0.4";
+  version = "0.0.7";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
   src = fetchFromGitHub {
     owner = "napari";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-aVdYOzkZ+dqB680oDjNCg6quXU+QgUZI09E/MSTagyA=";
+    rev = "refs/tags/v${version}";
```

Comment:
What's wrong with just `v${version}`?

Reply:
This helps to avoid branch/tag confusion and make the automated update approaches run smoother.

Reply:
I see. I don't recall seeing this in nixpkgs before, but it does sound good!